src/py42/_internal/client_factories.py

```diff
@@ -62,6 +63,7 @@ def __init__(
         self._detection_list_user_client = None
         self._ecm_session = None
         self._alert_rules_client = None
```

Comment:
add `self._file_event_session = None`  then set and return it (like we do with `ecm_session`) in `get_file_event_client()`

Reply:
Changed.